core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -256,7 +256,7 @@ class TransactionCoordinator(brokerId: Int,
 
       result match {
         case Left(err) =>
-          info(s"Returning $err error code to client for $transactionalId's AddPartitions request")
+          debug(s"Returning $err error code to client for $transactionalId's AddPartitions request")
```

Comment:
This becomes a very chatty line when you have a high transaction throughput. So I think debug is a more appropriate level.